### DISCUSSION TITLE
Glances docker install / shown tasks

### DISCUSSION THREAD

**Discussion by janreinhardt** [nicolargo/glances#2670] [Q&A] [answered]
Please, I am new to this Linux stuff, got myself a Raspi 5 for Christmas and started a big Docker + Portainer + Glances adventure 4 weeks ago:
I have set up Glances in two different ways. As a Docker stack via a pre-built Docker compose template. And by hand as a container via Portainer.
The one install gives me
`TASKS **284** (1008 thr),1 run,222 slp,0 oth Threads sorted automatically by CPU consumption`

the other shows
`TASKS **2** (17 thr),1 run,1 slp,0 oth Threads sorted automatically by CPU consumption`

I **think** this has to do with the "pid: host" setting? I have this set in the docker compose file, but how do I set this in Portainer? I have set the toggle: Runtime / Init in Portainer, but not with the desired effect? What am I doing wrong?
Id like to get rid of a "stack" of Glances, when I can run this as a standard container. But of course Id like Glances to show all the running tasks, not just 2.

**Comment by nicolargo** [comment, marked as answer]
Hi @janreinhardt !

unfortunatly, Portainer did not allow to access to the local process from container: https://github.com/portainer/portainer/issues/2431

Nicolas

**Reply by janreinhardt** [reply]
Thanks! So instead of using Portainer I started my Glances container from the CLI now 👍🏻
